src/main/java/com/denizenscript/depenizen/bukkit/events/shopkeepers/ShopKeeperTradeCompletedScriptEvent.java

```diff
@@ -0,0 +1,60 @@
+package com.denizenscript.depenizen.bukkit.events.shopkeepers;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.depenizen.bukkit.bridges.ShopkeepersBridge;
+import com.denizenscript.depenizen.bukkit.objects.shopkeepers.ShopKeeperTag;
+import com.nisovin.shopkeepers.api.events.ShopkeeperTradeCompletedEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class ShopKeeperTradeCompletedScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player completes shopkeeper trade
```

Comment:
It sounds like `shopkeeper trade` matches `ShopkeeperTradeCompletedEvent`
and `ShopkeeperTradeEvent` was reworked by the author to be a whole new/different initiated event - in Denizen scripts we should preserve the original functionality/intent rather than blindly matching the java code author's unusual changes.